source/documentation/enabling-migrations.html.md

```diff
@@ -58,6 +58,10 @@ To test that this is working, try ssh'ing from one compute node to another as th
       It worked!
       $
 
+In case you get a "This account is currently not available." error, you will need to enable logins to the "nova" user with:
+
+      # usermod -s /bin/bash nova
+
 ---> SELinux?
```

Comment:
Sure, of course. But that's kinda out of scope of this pull request. Perhaps you should make a separate one.